python/nav/web/crispyforms.py

```diff
@@ -69,59 +69,43 @@ class NumberField(forms.IntegerField):
 
 
 class FlatFieldset:
-    """
-    A class representing a fieldset for forms.
+    """A class representing a fieldset for forms.
     Only flat layout of children fields is supported out of the box.
     Any nesting of fields inside this fieldset might require custom
     class definitions akin to this one.
 
-    Attributes:
-        template (str): The path to the template used for rendering the fieldset.
-        Path is relative to the app's templates directory.
-        legend (str): The legend text for the fieldset.
-        fields (list): A list of fields to include in the fieldset.
-        css_class (str): Additional CSS classes to apply to the fieldset.
+    :cvar template: The path to the template used for rendering the fieldset. Path is relative to the app's templates directory.
```

Comment:
`cvar`?

Reply:
Aha.. I've never seen those before. Don't think they existed when we first began using Sphinx :)